src/modules/mc_pos_control/PositionControl.cpp

```diff
@@ -332,8 +332,8 @@ void PositionControl::updateConstraints(const vehicle_constraints_s &constraints
 	// constraints, then just use global constraints for the limits.
 
 	if (!PX4_ISFINITE(constraints.tilt)
-	    || !(constraints.tilt < math::max(_param_mpc_tiltmax_air.get(), _param_mpc_man_tilt_max.get()))) {
-		_constraints.tilt = math::max(_param_mpc_tiltmax_air.get(), _param_mpc_man_tilt_max.get());
+	    || !(constraints.tilt < math::radians(math::max(_param_mpc_tiltmax_air.get(), _param_mpc_man_tilt_max.get())))) {
+		_constraints.tilt = math::radians(math::max(_param_mpc_tiltmax_air.get(), _param_mpc_man_tilt_max.get()));
```

Comment:
Suggestion mainly for readability and avoiding duplication, put `math::radians(math::max(_param_mpc_tiltmax_air.get(), _param_mpc_man_tilt_max.get()))` into a const variable first.

Reply:
Thanks, I've made the changes.